### PR TITLE
moc-sdk changes for sfs download

### PR DESCRIPTION
**Pull request by cleonard-git**

### CODE REVIEW

File: services/compute/galleryimage/client.go [hunk 1]
```diff
@@ -39,10 +40,30 @@ func (c *GalleryImageClient) Get(ctx context.Context, location, name string) (*[
...
 }
 
 // Delete methods invokes delete of the compute resource
 func (c *GalleryImageClient) Delete(ctx context.Context, location, name string) error {
 	return c.internal.Delete(ctx, location, name)
 }
+
+// UploadImageFromLocal   methods invokes  UploadImageFromLocal  on the client
+func (c *GalleryImageClient) UploadImageFromLocal(ctx context.Context, location, imagePath, name string, compute *compute.GalleryImage) (*compute.GalleryImage, error) {
+	compute.SourceType = "local"
```
Comment (madhanrm) [inline services/compute/galleryimage/client.go:54]: can you define "local", "sfs" as enums in moc and reuse them. It is not dev friendly to hard code these here.

Comment (cleonard-git) [reply services/compute/galleryimage/client.go]: yeah I had that in the node agent code.  Brought it moc-sdk as well.

File: pkg/constant/constant.go [hunk 2]
```diff
@@ -11,4 +11,8 @@ const (
 	DefaultServerContextTimeout          = 10 * time.Minute
 	CertificateValidityThreshold float64 = (30.0 / 100.0)
 	RenewalBackoff               float64 = (2.0 / 100.0)
+	SFSSource                            = "sfs"
```
Comment (madhanrm) [inline pkg/constant/constant.go:14]: We could define these as .proto enums and get it auto generated, and used across all the layers.

Comment (cleonard-git) [reply pkg/constant/constant.go:14]: Done . https://github.com/microsoft/moc/pull/82

**Review by madhanrm — APPROVED** [review]

**Pull request merged**